src/components/SelectedAnnotation.jsx

```diff
@@ -203,7 +205,7 @@ class SelectedAnnotation extends React.Component {
     const notes = emptyText ? 'You cannot save an empty transcription.' : '';
     this.props.dispatch(togglePopup(
       <QuestionPrompt
-        confirm="Yes, delete"
+        confirm={this.props.translate('cribSheet.confirm')}
         deny="No, continue transcribing"
```

Comment:
Should this be localised too?

Reply:
We don't have a translation for that yet.

Reply:
I'd localise this in the view here and put placeholder strings in the locale files. No big deal, though, if you're aware that it still needs to be done.